app/src/main/java/com/nervousfish/nervousfish/activities/KeyManagementActivity.java

```diff
@@ -17,6 +45,67 @@
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_key_management);
+        this.setContentView(R.layout.activity_key_management);
+        final IServiceLocator serviceLocator = NervousFish.getServiceLocator();
+        final IDatabase database = serviceLocator.getDatabase();
+        String title = "";
+        String[] key = {};
+        try {
+            this.myProfile = database.getProfiles().get(0);
+            title = this.myProfile.getKeyPairs().get(0).getName();
+            key = this.myProfile.getKeyPairs().get(0).getPublicKey().getKey().split(" ");
```

Comment:
You can move these two statements outside the try-catch block and make the variables final